doc/codi.txt

```diff
@@ -264,6 +265,15 @@ g:codi#sync
 
              Default value is 0.
 
+                                                       *g:codi#use_buffer_dir*
+g:codi#use_buffer_dir
+             Whether or not switch to switch to the directory of the buffer
```

Comment:
definitely. i'll pull down your branch and install that in neovim here shortly and test it out. thanks much for taking the initiative to add this functionality (handy for us JS/node devs!)